### ISSUE TITLE
Fill `nm` and `nmctapp` gaps

### ISSUE THREAD

**Issue by grossir** [freelawproject/juriscraper#990]
Part of #929

### nm
Between 2019-04-11 and 2021-03-04 we have [6 opinions](https://www.courtlistener.com/?q=court_id%3Anm&type=o&order_by=dateFiled%20asc&stat_Precedential=on&stat_Non-Precedential=on&filed_after=04/11/2019&filed_before=03/04/2021). Should have [80](https://nmonesource.com/nmos/en/d/s/index.do?cont=&ref=&rdnpv=&rdnii=&rdnct=&d1=04%2F11%2F2019&d2=03%2F04%2F2021&ca=&p=&col=182&or=)

Update:
Part of this gap was patched by a merger (it has clusters), the new [gap](https://www.courtlistener.com/?q=court_id%3Anm&type=o&order_by=dateFiled%20desc&stat_Published=on&stat_Unpublished=on&filed_after=01%2F19%2F2020&filed_before=03%2F03%2F2021) has 0 opinions Filed After: 2020-01-19 › Filed Before: 2021-03-03 

Between 2021-04-13 and 2022-01-09 we have [0 opinions](https://www.courtlistener.com/?q=court_id%3Anm&type=o&order_by=dateFiled%20asc&stat_Precedential=on&stat_Non-Precedential=on&filed_after=04%2F13%2F2021&filed_before=01%2F09%2F2022), should[ have 30](https://nmonesource.com/nmos/en/d/s/index.do?cont=&ref=&rdnpv=&rdnii=&rdnct=&d1=04%2F13%2F2021&d2=01%2F09%2F2022&ca=&p=&col=182&or=).

### nmctapp
Between June 7th, 2019 and April 1st, 2022 we have [3 documents](https://www.courtlistener.com/?q=court_id%3Anmctapp&type=o&order_by=dateFiled%20asc&stat_Precedential=on&stat_Non-Precedential=on&filed_after=07%2F06%2F2019&filed_before=01%2F04%2F2022) in CL. From the [source](https://nmonesource.com/nmos/en/d/s/index.do?cont=&ref=&rdnpv=&rdnii=&rdnct=&d1=06%2F07%2F2019&d2=01%2F04%2F2022&ca=&p=&col=183&or=), we are missing 1265 documents

**Comment by grossir** [comment]
This source is using "Decisia", the same backend as `ark` sources #958 . An abstract scraper for this backend could be created, and each state source could inherit from it. Will leave that merging for a future issue

**Comment by grossir** [comment]
```
docker exec -it cl-django python /opt/courtlistener/manage.py cl_back_scrape_opinions --courts juriscraper.opinions.united_states.state.nm --backscrape-start=04/10/2019 --backscrape-end=03/05/2021

docker exec -it cl-django python /opt/courtlistener/manage.py cl_back_scrape_opinions --courts juriscraper.opinions.united_states.state.nm --backscrape-start=04/12/2021 --backscrape-end=01/10/2022

docker exec -it cl-django python /opt/courtlistener/manage.py cl_back_scrape_opinions --courts juriscraper.opinions.united_states.state.nmctapp --backscrape-start=07/05/2019 --backscrape-end=01/03/2022
```

**Comment by grossir** [comment]
the scraper server is blocked by decisia [too](https://github.com/freelawproject/juriscraper/issues/958#issuecomment-2819620693)

**Comment by flooie** [comment]
this appears to have been resolved? or we think we may have been unblocked.   Perhaps we can re-execute this code and move this forward?  same for ark?

**Comment by grossir** [comment]
Yes, we are unblocked. I will run the backscrapers very slowly. Currently running
```
nohup ./manage.py cl_back_scrape_opinions --courts juriscraper.opinions.united_states.state.nm --backscrape-start=2020/01/19 --backscrape-end=2021/04/03 --verbosity 3 --days-interval=7 --backscrape-wait=180 > /tmp/log_nm2020_2021.txt &
```



Now running
```
nohup ./manage.py cl_back_scrape_opinions --courts juriscraper.opinions.united_states.state.nmctapp --backscrape-start=2019/07/05 --backscrape-end=2022/01/03 --verbosity 3 --days-interval=7 --backscrape-wait=180 > /tmp/log_nmctapp2019_2022.txt &
```


and


```
 nohup ./manage.py cl_back_scrape_opinions --courts juriscraper.opinions.united_states.state.nm --backscrape-start=2021/04/13 --backscrape-end=2022/01/09 --verbosity 3 --days-interval=10 --backscrape-wait=300 > /tmp/log_nm2021_2022.txt &
```

**Comment by grossir** [comment]
The gaps in the first comment are filled
